src/main/java/mate/academy/model/Movie.java

```diff
@@ -0,0 +1,50 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "movies")
+public class Movie {
+    private Long id;
+    private String title;
+    private String description;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
put these lines above the id field

Reply:
Lets remove this annotations `@Id and @GeneratedValue(strategy = GenerationType.IDENTITY)` over `getId()` method and leave them only above `private Long id;` parameter